TAF/testScenarios/functionalTest/V2-API/support-notifications/subscription/GET-Negative.robot

```diff
@@ -0,0 +1,90 @@
+${url}            ${supportNotificationsUrl}
+${api_version}    v2
+
+*** Test Cases ***
+# /subscription/all
+ErrSubscriptionGET001 - Query all subscriptions with non-int value on offset
+    When Query All Subscriptions With Invalid Offset
+    Then Should Return Status Code "400"
+    And Should Return Content-Type "application/json"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+
+ErrSubscriptionGET002 - Query all subscriptions with non-int value on limit
+    When Query All Subscriptions With Invalid Limit
+    Then Should Return Status Code "400"
+    And Should Return Content-Type "application/json"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+
+# /subscription/category/{category}
+ErrSubscriptionGET003 - Query subscriptions with non-existed category
+    When Query Subscriptions With Non-existed Category
+    Then Should Return Status Code "404"
+    And Should Return Content-Type "application/json"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+
+ErrSubscriptionGET004 - Query subscriptions with specified category and non-int value on offset
+    When Query Subscriptions With Specified Category And Invalid Offset
+    Then Should Return Status Code "400"
+    And Should Return Content-Type "application/json"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+
+ErrSubscriptionGET005 - Query subscriptions with specified category and non-int value on limit
+    When Query Subscriptions With Specified Category And Invalid Limit
+    Then Should Return Status Code "400"
+    And Should Return Content-Type "application/json"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+
+# /subscription/label/{label}
+ErrSubscriptionGET006 - Query subscriptions with non-existed label
+    When Query Subscriptions With Non-existed Label
+    Then Should Return Status Code "404"
+    And Should Return Content-Type "application/json"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+
+ErrSubscriptionGET007 - Query subscriptions with specified label and non-int value on offset
+    When Query Subscriptions With Specified Label And Invalid Offset
+    Then Should Return Status Code "400"
+    And Should Return Content-Type "application/json"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+
+ErrSubscriptionGET008 - Query subscriptions with specified label and non-int value on limit
+    When Query Subscriptions With Specified Label And Invalid Limit
+    Then Should Return Status Code "400"
+    And Should Return Content-Type "application/json"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+
+# /subscription/receiver/{receiver}
+ErrSubscriptionGET009 - Query subscriptions with non-existed receiver
```

Comment:
Shouldn't this case return 200 and empty array?

Reply:
Removed.